code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+			i--
+
+//////////////////////////////////////////////////////////////////////////////////
+
+// TSCs
+/datum/lore/organization/tsc/nanotrasen
+	name = "Nanotrasen Incorporated"
+	short_name = "Nanotrasen"
+	acronym = "NT"
+	desc = "Nanotrasen is one of the foremost research and development companies in SolGov space. \
+	Originally focused on consumer products, their swift move into the field of Plasma has lead to \
+	them being the foremost experts on the substance and its uses. In the modern day, Nanotrasen prides \
+	itself on being an early adopter to as many new technologies as possible, often offering the newest \
+	products to their employees. In an effort to combat complaints about being 'guinea pigs', Nanotrasen \
+	also offers one of the most comprehensive medical plans in SolGov space, up to and including cloning \
+	and therapy.\
+	<br><br>\
+	NT's most well known products are its plasma based creations, especially those used in Cryotherapy. \
+	It also boasts an prosthetic line, which is provided to its employees as needed, and is used as an incentive \
+	for newly tested posibrains to remain with the company."
+	history = "" // To be written someday.
+	work = "research giant"
+	headquarters = "Earth"
+	motto = ""
+
+	ship_prefixes = list("NSV" = "exploration", "NTV" = "hauling", "NDV" = "patrol", "NRV" = "emergency response")
+	// Note that the current station being used will be pruned from this list upon being instantiated
+	destination_names = list(
+		"NSS Exodus in Nyx",
+		"NCS Northern Star in Vir",
+		"NCS Southern Cross in Vir",
+		"NDV Icarus in Nyx",
+		"an asteroid orbiting Kara",
+		"an asteroid orbiting Rota",
+		)
+
+/datum/lore/organization/tsc/nanotrasen/New()
+	..()
+	spawn(1) // BYOND shenanigans means using_map is not initialized yet.  Wait a tick.
+		// Get rid of the current map from the list, so ships flying in don't say they're coming to the current map.
+		var/string_to_test = "[using_map.station_name] in [using_map.starsys_name]"
+		if(string_to_test in destination_names)
+			destination_names.Remove(string_to_test)
+
+
+
+/datum/lore/organization/tsc/hephaestus
+	name = "Hephaestus Industries"
+	short_name = "Hephaestus"
+	acronym = "HI"
+	desc = "Hephaestus Industries is the largest supplier of arms, ammunition, and small military vehicles in Sol space. \
+	Hephaestus products have a reputation for reliability, and the corporation itself has a noted tendency to stay removed \
+	from corporate politics. They enforce their neutrality with the help of a fairly large asset-protection contingent which \
+	prevents any contracting polities from using their own materiel against them. SolGov itself is one of Hephastus' largest \
+	bulk contractors owing to the above factors."
+	history = ""
+	work = "arms manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("HTV" = "freight", "HTV" = "munitions resupply")
+	destination_names = list(
+		"a SolGov dockyard on Luna"
+		)
+
```

Comment:
/datum/lore/organization/tsc/waffle
	name = "Waffle Corporation"
	short_name = "Waffle Co."
	acronym = "WC"
	desc = "The once prominent competitor of Donk Corporation, Waffle Co. is well-known for its popular line of Waffle Co. brand waffles and their use of violent tactics against competitors - often bribing, extorting, blackmailing or sabotaging businesses that pose a direct or indirect threat to their market share. They have recently fallen on hard times primarily due to to severe mismanagement which lead to much of their private arsenal being swindled by a pirate faction known as the Gorlex Marauders. Waffle Co. commonly engages in smear campaigns against Donk Co., maintaining that the original recipe for Donk Pockets was stolen from them."
	history = ""
	work = "food logistics and marketing firm"
	headquarters = ""
	motto = ""

	ship_prefixes = list("W-Co." = "transportation")
	destination_names = list()